README.md

```diff
@@ -15,3 +15,6 @@ $ java Calculate <action-name> <arguments...>
 - `multiply A B` - Multiplies the `A` by `B`
 - `divide A B` - Divides `A` by `B`
 - `mod A B` - Performs modulo operations from integer `A` and `B`
+- `sin rad` - Gets the sin of the input `rad`
+- `cos rad` - Gets the cos of the input `rad`
+- `tan rad` - Gets the tan of the input `rad`
```

Comment:
I think the description could be a bit clearer in my opinion. Like, "Gets the sin of the inputted value in radians". I think this way "**rad**" can be better understood.